scss/_variables.scss

```diff
@@ -1250,6 +1247,21 @@ $navbar-brand-logo-transition:         width $navbar-transition-duration $navbar
 $navbar-active-transition:             bottom $navbar-transition-duration $navbar-transition-timing-function !default;
 // End mod
 
+// Boosted mod: Orange navbar
+// scss-docs-start orange-navbar-variables
+$navbar-orange-padding-x:              .125rem !default;
```

Comment:
`$navbar-orange-*` or `$navbar-supra-*` choose the same basis

Reply:
In fact navbar-orange variables were made in anticipation of being reused in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/727

navbar-supra were IMO "exclusive" variables for supra bars. 

Maybe I should rename everything navbar-orange indeed

Reply:
renamed navbar-orange-supra ...